### DISCUSSION TITLE
How to get current mapleader `echo mapleader` not work.

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#1834] [Q&A] [answered]
Neither `echo mapleader` nor `lua print(vim.g.mapleader)` work. How to fix it?

**Comment by dpetka2001** [comment, marked as answer]
I'd guess it's because it's set to space. If you do `echo maplocalleader` it will correctly output `\` or if you change the `mapleader` in your `options.lua` to something else (I tried with `;`) it will correctly show what it is.
